apps/sequence/base.de.i18n

```diff
@@ -1,22 +1,22 @@
 SequenceApp = "Folge"
 SequenceAppCapital = "FOLGE"
 SequenceTab = "Folgen"
-AddSequence = "Folge hinzuzufugen"
-ChooseSequenceType = "Das Bildungsgesetz der Folge auswahlen"
+AddSequence = "Folge hinzufuegen"
+ChooseSequenceType = "Das Bildungsgesetz der Folge auswaehlen"
 SequenceType = "Bildungsgesetz der Folge"
 Explicit = "Explizit"
-SingleRecurrence = "Rekursiv, Ordnung 1"
-DoubleRecurrence = "Rekursiv, Ordnung 2"
-SequenceOptions = "Optionen Folge"
+SingleRecurrence = "Rekursion 1. Ordnung"
+DoubleRecurrence = "Rekursion 2. Ordnung"
+SequenceOptions = "Optionen der Folge"
 SequenceColor = "Farbe der Folge"
-DeleteSequence = "Folge loschen"
+DeleteSequence = "Folge loeschen"
 NoSequence = "Keine Folge"
 NoActivatedSequence = "Keine aktive Folge"
 NStart = "Startwert"
 NEnd = "Endwert"
-TermSum = "Summe der Glieder"
-SelectFirstTerm = "Erster Glied "
-SelectLastTerm = "Letztes Glied "
-ValueNotReachedBySequence = "Der Wert wird nicht von der Folge erreicht"
-NColumn = "n Spalte"
+TermSum = "Summe der Bedingungen"
```

Comment:
I am pretty sure the translation of "term" as "Glied" is correct, if this is about terms as parts of mathematical sequences ("Folgen"). So, the translations here should indeed be
"Summe der Glieder"
"Erstes Glied "
"Letztes Glied "
or, also correct,
"Summe der Terme"
"Erster Term "
"Letzter Term ".